src/Randomizer.SMZ3.Tracking/TrackerServiceCollectionExtensions.cs

```diff
@@ -20,45 +22,37 @@ public static class TrackerServiceCollectionExtensions
         /// <summary>
         /// Adds the services required to start using Tracker.
         /// </summary>
-        /// <typeparam name="TWorldAccessor">
-        /// A type implementing <see cref="IWorldAccessor"/>. The type should
-        /// already be registered as a singleton.
-        /// </typeparam>
         /// <param name="services">
         /// The service collection to add Tracker to.
         /// </param>
         /// <returns>A reference to <paramref name="services"/>.</returns>
-        public static IServiceCollection AddTracker<TWorldAccessor>(this IServiceCollection services)
-            where TWorldAccessor : class, IWorldAccessor
+        public static IServiceCollection AddTracker(this IServiceCollection services)
         {
             services.AddBasicTrackerModules<TrackerModuleFactory>();
             services.AddScoped<TrackerModuleFactory>();
 
-            services.AddSingleton<IWorldAccessor>(x => x.GetRequiredService<TWorldAccessor>());
             services.AddSingleton<TrackerConfigProvider>();
             services.AddSingleton<IHistoryService, HistoryService>();
-            services.AddScoped(serviceProvider =>
+            services.AddSingleton(serviceProvider =>
```

Comment:
This introduces an issue where if you open tracker, track items, close tracker, then open a new instance of tracker for a seed without a save state, the tracked items will persist into the new tracker instance.

Reply:
I'll add a test for this, too.

Reply:
Actually, considering I want most of the state to be in services anyway, I might just remove these and force use of the ConfigProvider instead, so that we won't depend on the config classes themselves holding the state.